test/C/src/token/include/array.h

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
Does this file belong here, or would it be better to make it available through reactor-c?

Reply:
This is only a test example, not a thoroughly thought-through array implementation in C, so I think it's better to leave here.